### PR TITLE
layers: Fix VUID churn for 268

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Fixing VUID found after 268 header update

### CODE REVIEW

File: layers/core_checks/cc_copy_blit_resolve.cpp [hunk 1]
```diff
@@ -3373,7 +3373,7 @@ bool CoreChecks::ValidateCmdBlitImage(VkCommandBuffer commandBuffer, VkImage src
                                              region.dstSubresource.layerCount, dst_subresource_loc, vuid);
         // Check that src/dst layercounts match
         if (region.srcSubresource.layerCount != region.dstSubresource.layerCount) {
-            vuid = is_2 ? "VUID-VkImageBlit2-layerCount-00239" : "VUID-VkImageBlit-layerCount-00239";
+            vuid = is_2 ? "VUID-VkImageBlit2-layerCount-08800" : "VUID-VkImageBlit-layerCount-08800";
```
Comment (arno-lunarg) [inline layers/core_checks/cc_copy_blit_resolve.cpp:3376]: VUID:
> If neither of the layerCount members of srcSubresource or dstSubresource are VK_REMAINING_ARRAY_LAYERS, the layerCount members of srcSubresource or dstSubresource must match

I did not see any check for `VK_REMAINING_ARRAY_LAYERS` in this function, is it missing or is it done elsewhere?

File: layers/core_checks/cc_copy_blit_resolve.cpp [hunk 2]
```diff
@@ -3695,7 +3695,7 @@ bool CoreChecks::ValidateCmdResolveImage(VkCommandBuffer commandBuffer, VkImage
         // layer counts must match
         if (src_subresource.layerCount != dst_subresource.layerCount) {
             const LogObjectList objlist(commandBuffer, srcImage, dstImage);
-            vuid = is_2 ? "VUID-VkImageResolve2-layerCount-00267" : "VUID-VkImageResolve-layerCount-00267";
+            vuid = is_2 ? "VUID-VkImageResolve2-layerCount-08803" : "VUID-VkImageResolve-layerCount-08803";
```
Comment (arno-lunarg) [inline layers/core_checks/cc_copy_blit_resolve.cpp:3698]: Same comment here

**Review by arno-lunarg — APPROVED** [review]
LGTM, 2 questions

File: layers/core_checks/cc_copy_blit_resolve.cpp (hunk 1 above)
Comment (spencer-lunarg) [reply layers/core_checks/cc_copy_blit_resolve.cpp:3376]: yep was going to comment on this, this is being done in the followup PR as a "fix to a bad VU", goal of this was to just remove the old-hard-to-find in the spec VUID number

good question though

**Pull request merged**